### PR TITLE
fix(workflow): Dim suggested avatar (WOR-543)

### PR DESCRIPTION
**Pull request by scttcper**
Before:
image

After:
image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/avatar/styles.tsx [hunk 1]
```diff
@@ -13,10 +11,13 @@ export const imageStyle = (props: ImageStyleProps) => css`
   top: 0px;
   left: 0px;
   border-radius: ${props.round ? '50%' : '3px'};
-  border: ${props.suggested ? `1px dashed ${theme.gray400}` : 'none'};
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/components/avatar/styles.tsx:16]: hmmm does removing this also remove the dashed border from a suggested team assignee?

Comment (scttcper) [reply src/sentry/static/sentry/app/components/avatar/styles.tsx:16]: it is moved to the parent so that the css filters do not apply to it

Comment (davidenwang) [reply src/sentry/static/sentry/app/components/avatar/styles.tsx:16]: oooh my bad for some reason I thought the baseavatar stuff only applied to user avatars.

File: src/sentry/static/sentry/app/components/avatar/styles.tsx [hunk 2]
```diff
@@ -13,10 +11,13 @@ export const imageStyle = (props: ImageStyleProps) => css`
   top: 0px;
   left: 0px;
   border-radius: ${props.round ? '50%' : '3px'};
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/components/avatar/styles.tsx:13]: since `border-radius` is moved to the parent can we remove it now?

**Review by davidenwang — APPROVED** [review]

File: src/sentry/static/sentry/app/components/avatar/styles.tsx (hunk 2 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/components/avatar/styles.tsx:13]: i tried it, but the img avatars stay square and stick out of the container

Comment (davidenwang) [reply src/sentry/static/sentry/app/components/avatar/styles.tsx:13]: oof that sucks, aight no worries

**Pull request merged**